knodle/trainer/trainer.py

```diff
@@ -79,6 +80,19 @@ def initialise_optimizer(self):
 
 class BaseTrainer(Trainer):
 
+    def __init__(
+            self,
+            model: Module,
+            mapping_rules_labels_t: np.ndarray,
+            model_input_x: TensorDataset,
+            rule_matches_z: np.ndarray,
+            **kwargs):
+        if kwargs.get("trainer_config", None) is None:
+            kwargs["trainer_config"] = BaseTrainerConfig()
+        super().__init__(model, mapping_rules_labels_t, model_input_x, rule_matches_z, **kwargs)
+
+        check_other_class_id(self.trainer_config, self.mapping_rules_labels_t)
```

Comment:
I don't like the naming of the function 100% yet because without looking at the code I'm not 100% sure what it does. Maybe 
`check_valitd_other_class_id`
Also we should discuss, whether we stick with other_class or negative class, which is often used in docs.